src/components/home/modals/Settings.vue

```diff
@@ -127,9 +128,23 @@ export default {
             ]
         }
     },
+    computed: {
+        userInfo() {
+            return JSON.parse(window.localStorage.getItem(this.defaultAddress))
+        },
+        defaultAddress() {
+            return Vue.ls.get('address')
+        },
+        seedAddress() {
+            if (Vue.ls.get('address')) {
+                return Vue.ls.get('address')
+            }
+        }
+    },
     methods: {
         changeSession() {
-            this.setUsrLocalStorage('sessionTimeout', this.selectedSession)
+            Vue.set(this.userInfo, 'sessionTimeout', this.selectedSession)
```

Comment:
Should not set `this.userInfo` here. This will make userInfo not update. The userInfo should get from local storage every time before set.

Reply:
already get userInfo from the newest local storage and change it in other components